packages/akashic-cli-lib-manage/src/install/install.ts

```diff
@@ -111,6 +111,28 @@ export function promiseInstall(param: InstallParameterObject): Promise<void> {
 			if ((normalizedParam.plugin != null) && conf.findExistingOperationPluginIndex(normalizedParam.plugin) !== -1)
 				throw new Error("Conflicted code for operation plugins: " + normalizedParam.plugin + ".");
 
+			if (normalizedParam.useMmp && conf._content.moduleMainScripts != null)
+				throw new Error(
+					"To enable the `--use-mmp` option, remove the `moduleMainScripts` property. " +
+					"This issue may be resolved by running the following command: \n" +
+					"> akashic scan globalScripts --use-mmp"
```

Comment:
いくつか気になりました:
- ```remove the `moduleMainScripts` property``` という指示に従ってゲーム開発者が単に削除すると、ゲームが動かなくなることがありそう
- ゲーム開発者が mmp に移行したいかどうかは状況によりそう
- akashic scan globalScripts が modulesMainScripts とどう関係あるのか示されていない

エラーメッセージは (a) 何が起きたか、(b) 何が問題か (自明でないなら)、(c) 対処方法・必要な作業 の順で示せるとよいように思え、こんな感じでどうでしょう。

```
An old property `moduleMainScripts` found in game.json and rejected by `--use-mmp`
option that forces to use `moduleMainPaths` instead. If you need `moduleMainScripts`
(e.g. to support older Akashic Engine), remove `--use-mmp`. Otherwise run the
following command to migrate to `moduleMainPaths`:
```

Reply:
[b280474](https://github.com/akashic-games/akashic-cli/pull/1353/commits/b2804742bcf95908f2ebe1714474dd5d1bd260df) そのあたりの温度感は私の方では判断できかねるので提示いただいた文言に修正しました。